src/app/analytics-handler.ts

```diff
@@ -66,6 +66,7 @@ class Analytics {
      * @param eventData {IAnalyticsData}
      */
     public track(eventData: IAnalyticsData): void {
+        console.warn('sending analytics data', eventData);
```

Comment:
this is supposed to be a logger right? Not console log?

Reply:
God darnit, not even a logger but was used during debugging....